cmd/scan.go

```diff
@@ -100,7 +102,13 @@ func (s *fileScanner) Do(path interface{}, ds *utils.DoerState) string {
 	}
 	defer f.Close()
 
-	analysis, err := s.scanner.ScanFile(f, progressCh)
+	var analysis *vt.Object
+	if s.hasPassword {
```

Comment:
why not simply using `s.hasPassword != ""` instead of having a separate field `hasPassword`?

Reply:
I was thinking in the case of accepting empty passwords, but you're right let's simplify it.